README.rst

```diff
@@ -1108,6 +1108,22 @@ users (those whose User.type is 'rodsadmin' or 'rodsuser'). The empty group ('em
 members, so it doesn't show up in our final list.
 
 
+iRODS Permissions (ACLs)
+------------------------
+
+The :code:`iRODSAccess` class offers a convenient dictionary interface mapping iRODS permission
+strings to the corresponding integer codes:
+
+>>> from irods.access import iRODSAccess
+>>> iRODSAccess.keys()
+['null', 'read_metadata', 'read_object', 'create_metadata', 'modify_metadata', 'delete_metadata', 'create_object', 'modify_object', 'delete_object', 'own']
```

Comment:
it's true that `.keys()` won't return 'read' and 'write'?   that's fine/good... just confirming.

Reply:
that's right... iRODS seems to prefer this also when returning permissions from acls.get(...).  It will accept the shorter forms in acls.set(...) . So 'write' or 'modify_object' will both work for that latter case.

Reply:
gotcha - thanks.  i like the 'accept' the old things, but report the 'more correct/specific' new things.

Reply:
Oh... slight modification of what I said, above. set is more restrictive than I though ...  for example in 4.2.x `acls.set` will  accept 'write' but not `modify object'; however acls.get  returns `modify object`. And it's the server doing it.

Reply:
@trel yep I think @mstfdkmn was right, this is a good convenience to have around.